prov/efa/src/efa.h

```diff
@@ -558,4 +559,13 @@ static inline bool efa_is_cache_available(struct efa_domain *efa_domain)
 	return efa_domain->cache;
 }
 
+enum efa_mem_monitor_type {
+        EFA_UFFD_MONITOR,
+        EFA_MEMHOOKS_MONITOR
+};
+static const char * const efa_mem_monitors[] = {
+        [EFA_UFFD_MONITOR] = "uffd",
```

Comment:
Do we actually need a constant, or do we just need a string -> function pointer table?  Seems like the latter could save some code complexity.